examples/seq2seq/README.md

```diff
@@ -0,0 +1,75 @@
+# Sequence-to-sequnce learning example for machine translation
+
+This is a minimal example of sequence-to-sequence learning. Sequence-to-sequence learning model is a conversion model from a sequence to a sequence. You can regard many tasks in natural language processing field as this type of task, such as machine translation, dialogue and summarization.
```

Comment:
Consider rewording:

This is a minimal example of sequence-to-sequence learning. Sequence-to-sequence is a learning model that converts an input sequence into an output sequence. You can regard many tasks in the natural language processing field as this type of task, such as machine translation, dialogue and summarization.